app/views/Export/ExportSelectHA.js

```diff
@@ -0,0 +1,164 @@
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  ActivityIndicator,
+  StatusBar,
+  StyleSheet,
+  TouchableHighlight,
+  View,
+} from 'react-native';
+import { FlatList } from 'react-native-gesture-handler';
+import { SafeAreaView } from 'react-native-safe-area-context';
+import { SvgXml } from 'react-native-svg';
+
+import { Icons } from '../../assets';
+import { Button } from '../../components/Button';
+import { IconButton } from '../../components/IconButton';
+import { Typography } from '../../components/Typography';
+import Colors from '../../constants/colors';
+import fontFamily from '../../constants/fonts';
+import { Theme } from '../../constants/themes';
+import { HCAService } from '../../services/HCAService';
+
+const getParsedAuthorities = async () => {
+  const authorities = await HCAService.getAuthoritiesList();
+  return authorities.map(authority => ({
+    name: Object.keys(authority)[0],
+    url: Object.values(authority)[0][0].url,
+  }));
+};
+
+export const ExportSelectHA = ({ navigation }) => {
+  const { t } = useTranslation();
+
+  const [authorities, setAuthorities] = useState(null);
+  const [selectedAuthority, setSelectedAuthority] = useState(null);
+  useEffect(() => {
+    getParsedAuthorities().then(setAuthorities);
+  }, []);
+
+  const toggleSelected = HA => {
+    if (HA === selectedAuthority) {
+      setSelectedAuthority(null);
+    } else {
+      setSelectedAuthority(HA);
+    }
+  };
+
+  return (
+    <Theme use='default'>
+      <StatusBar
+        barStyle='dark-content'
+        backgroundColor={Colors.VIOLET_BUTTON}
+        translucent={false}
+      />
+      <View style={styles.wrapper}>
+        <SafeAreaView style={{ flex: 1, paddingBottom: 0 }}>
```

Comment:
The version of `SafeAreaView` we're using requires doing this double nesting of views.